app/controllers/devise/hitobito/passwords_controller.rb

```diff
@@ -19,7 +19,11 @@ def successfully_sent?(resource)
   def create
     previous_locale = I18n.locale
     resource = resource_class.find_by(email: resource_params["email"])
-    I18n.locale = resource&.language || previous_locale
+    I18n.locale = if Settings.application.languages.to_h.key?(resource&.language&.to_sym)
```

Comment:
```suggestion
    I18n.locale = if Settings.application.languages.keys.include?(resource&.language&.to_sym)
```

But doesn't matter much.